src/components/navigation/header/config/HeaderNavigationConfig.tsx

```diff
@@ -95,17 +104,40 @@ export class HeaderNavigationConfig extends BaseConfig<HeaderNavigationConfigIns
     return this.mappedConfig;
   }
 
+  replacePathParams = (
+    link: LocalizedLinks | undefined
+  ): LocalizedLinks | undefined => {
+    if (link && this.urlPathParams) {
+      const transformedLink = Object.entries(link).reduce(
+        (acc, [key, value]) => {
+          return {
+            ...acc,
+            [key]: replaceParameters({
+              path: value,
+              params: this.urlPathParams,
+            }),
+          };
+        },
+        {} as LocalizedLinks
```

Comment:
I think reduce takes a generic so that you don't have to cast it. Not 100% sure though
`.reduce<LocalizedLinks>(...)`

Reply:
Doesn't work unfortunately :(